app/assets/javascripts/latlong_map.coffee

```diff
@@ -3,38 +3,61 @@ marker = null
 # fallback to PVTA if we can't geocode the incident's location
 PVTA = { lat: 42.105552, lng: -72.596511 }
 
-createMap = (mapSelector, latLng, placeInitialMarker) ->
+createMap = (mapSelector, latLng, newValues) ->
   map = new google.maps.Map mapSelector[0], { zoom: 15, tilt: 0, center: latLng }
-  if placeInitialMarker
-    placeMarker map, latLng
+  if newValues
+    placeMarker map, latLng, mapSelector, true
+  else
+    placeMarker map, latLng, mapSelector, false
   google.maps.event.addListener map, 'click', (event) ->
-    placeMarker map, event.latLng
-    fillLatLngFields event.latLng
+    latLong = { lat: event.latLng.lat(), lng: event.latLng.lng() }
```

Comment:
Isn't this just `latLong = event.latLng`? Or does `event.latLng` have other properties that we don't care about?

Reply:
event.latLng has other properties. Using event.latLng works for placing the marker on the map but I need to use .lat() in order to fill in the lat form field with a number.